src/ios/CFCallNumber.m

```diff
@@ -1,34 +1,24 @@
 #import <Cordova/CDVPlugin.h>
 #import "CFCallNumber.h"
-
+ 
 @implementation CFCallNumber
-
+ 
 - (void) callNumber:(CDVInvokedUrlCommand*)command {
-
+ 
     [self.commandDelegate runInBackground:^{
-
+ 
         CDVPluginResult* pluginResult = nil;
         NSString* number = [command.arguments objectAtIndex:0];
-        number = [number stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
-
-        if( ! [number hasPrefix:@"tel:"]){
-            number =  [NSString stringWithFormat:@"tel:%@", number];
-        }
-
-        if(![[UIApplication sharedApplication] canOpenURL:[NSURL URLWithString:number]]) {
```

Comment:
Oops, i forked the repo and pulled to the wrong place, sorry about that. 

The thing is that the plugin stops at this line 
`number = [number stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding];`
and at all the error handling stuff.

I couldn't find the problem myself, i can't programm objective-c, so all i've done is concat "tel:" to the number you get from the javascript and make the phonecall with that.